conda/api.py

```diff
@@ -1,5 +1,110 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from .core.solve import Solver  # NOQA
-Solver = Solver  # lgtm [py/redundant-assignment]
+from .common.constants import NULL
+from .core.package_cache_data import PackageCacheData as _PackageCacheData
+from .core.prefix_data import PrefixData as _PrefixData
+from .core.solve import Solver as _Solver, DepsModifier as _DepsModifier
+from .core.subdir_data import SubdirData as _SubdirData
+from .models.channel import Channel
+
+
+DepsModifier = _DepsModifier
+
+
+class Solver(object):
```

Comment:
We need a docstring here, that also explains why this `Solver` wrapper is needed, or intended for?

Reply:
Any recommendation on how to do this without duplicating everything that's already in the underlying `Solver` class?  Plenty of nice docstrings here: https://github.com/conda/conda/blob/master/conda/core/solve.py#L52